chapter01/Intro_example_short.ipynb

```diff
@@ -0,0 +1,604 @@
+    "\n",
+    "\n",
+    "## <font color=#CD853F> Contents of this notebook: </font>\n",
+    "  * [0. Learning objectives](#r-learning-objectives)\n",
+    "  * [1. Defining the BIP](#r-defining-the-bip)\n",
+    "  * [2. Solving the BIP](#r-solving-the-bip)\n",
+    "  * [3. Summary](#r-summary)\n",
+    "  * [References](#r-references)\n",
+    "\n",
+    "\n",
+    "## <font color=#CD853F> 0. Learning objectives </font> <a class=\"anchor\" id=\"r-learning-objectives\"></a>\n",
+    "  * Create a simple BIP in CUQIpy\n",
+    "      * Create a linear forward model object\n",
+    "      * Create distribution objects that represent the prior, the noise, and the data distributions\n",
+    "      * Use the `BayesianProblem` class to define the BIP\n",
+    "\n",
+    "  * Solve a simple BIP in CUQIpy\n",
+    "      * Compute the maximum a posteriori (MAP) estimate\n",
+    "      * Sample from a simple posterior distribution and visualize the results"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## <font color=#CD853F> 1. Defining the BIP </font> <a class=\"anchor\" id=\"r-defining-the-bip\"></a>\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Consider the following inverse problem: given observed data $b$, determine $x_1$, and $x_2$:\n",
+    "\n",
+    "$$b = x_1 + x_2 + e \\;\\;\\mathrm{with}\\;\\; e \\sim \\mathrm{Gaussian}(0, 0.1)$$\n",
+    "\n",
+    "We can also write it in the following matrix form:\n",
+    "\n",
+    "$$b = \\mathbf{A}\\mathbf{x} + e = \\large(1,1\\large)\\binom{x_1}{x_2} + e$$\n",
+    "\n",
+    "\n",
+    "|variable     |    description          |dimension      |\n",
+    "|:------------|:------------------------|:--------------|\n",
+    "|$\\mathbf{x}$ |parameter to be inferred |2-dimensional​  |\n",
+    "|$\\mathbf{A}$ |forward model            |1-by-2 matrix  |\n",
+    "|$b$          |data                     | 1-dimensional |\n",
+    "|$e$          |noise                    |1-dimensional  ​|\n",
+    "\n",
+    "This problem is:\n",
+    "* Considered a linear inverse problem since the forward model is linear.\n",
+    "* Ill-posed (in the sense of Hadamard [1]) since the solution is not unique, i.e., for some given value of $b$, e.g., $b=3$, all parameter pairs $(x_1, x_2)$ satisfying $x_1 + x_1 = 3$ are solutions to the (noise-free) problem."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let us define the BIP components and solve it using CUQIpy."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "hide-cell"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# Importing the required libraries\n",
+    "from cuqi.distribution import Gaussian\n",
+    "from cuqi.problem import BayesianProblem\n",
+    "from cuqi.model import LinearModel\n",
+    "from cuqi.utilities import plot_1D_density, plot_2D_density\n",
```

Comment:
The rendered notebook on the link you provide throws an error here. Unable to import plot_1D_density.

Reply:
Thanks @nabriis 

You can see this fixed in https://amal-ghamdi.github.io/CUQI-Book-test/chapter01/Intro_example_short.html (in the test book)

I added --pre in the requirements file to use the pre-release and now no such error